### PR TITLE
Update to dev setup, and added logging to aid in debugging Facebook token issue (/auth/facebook/token)

### PR DESCRIPTION
**Pull request by tanderegg**
/auth/facebook/token?access_token=XXXX does not work in Staging and Production.  This PR adds additional logging, while also fixing some issues with the local setup.  Endpoint works when running locally, so debugging in Staging is required.

### CODE REVIEW

**Comment by tanderegg** [comment]
@solomon-negusse Reminder to take a look at this when you can, thanks!

**Comment by solomon-negusse** [comment]
>@solomon-negusse Reminder to take a look at this when you can, thanks!

Thanks for the reminder, @tanderegg. Will check it out tomorrow.

File: docker-compose-develop.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
   develop:
     build: .
     ports:
-      - "30505:30505"
+      - "9000:9000"
```
Comment (solomon-negusse) [inline docker-compose-develop.yml:6]: Localstack looks for the MS on [30505](https://github.com/resource-watch/api-infrastructure/blob/production/terraform-k8s-infrastructure/modules/k8s_microservice_routing/microservices/authorization/main.tf#L14) for dependent microservices to access it although 30505:30505 probably never worked as the Dockerfile only exposes 9000.  Setting this to 30505:9000 would work with localstack and mimick the cloud setup.

**Comment by solomon-negusse** [comment]
@tanderegg left one comment, LGTM otherwise.

**Review by gtempus — APPROVED** [review]
I walked through this while pairing with @solomon-negusse today. Thanks for all of the improvements! 💪

**Pull request merged**